train.py

```diff
@@ -20,12 +23,30 @@ def train(hparams) -> None:
     Args:
         hparams (argparse.Namespace): Stores all passed args
     """
-    if hparams.debug is True:
+    if hparams.debug:
         hparams.logger = None
         profiler = True
     else:
         hparams.logger = WandbLogger(project=hparams.project)
+        hparams.logger.experiment
```

Comment:
was this left here by mistake?

Reply:
No. I wanted to save the json file under the wandb.run.id name. But to access that I must give time to wandb to start up and choose a name. This line does that. It just starts wandb now rather than when the Trainer starts.